src/Model/User.php

```diff
@@ -96,6 +96,59 @@ class User
 
 	private static $owner;
 
+	/**
+	 * Sends a welcome e-mail to the specified user
+	 *
+	 * @param array $user The user
+	 * @param string $password The user's password
+	 * @param string $preamble The user's password
+	 *
+	 * @return NULL|boolean from notification() and email() inherited
+	 * @throws HTTPException\InternalServerErrorException
+	 */
+	private static function sendWelcomeEmail(array $user, string $password, string $preamble)
+	{
+		$body = DI::l10n()->t( <<<'EOT'
+			The login details are as follows:
+
+			Site Location: 1$s
+			Login Name:    %2$s
+			Password:      %3$s
+
+			You may change your password from your account "Settings" page after logging
+			in.
+
+			Please take a few moments to review the other account settings on that page.
+
+			You may also wish to add some basic information to your default profile
+			' . "\x28" . 'on the "Profiles" page' . "\x29" . ' so that other people can easily find you.
+
+			We recommend adding a profile picture, adding some profile "keywords" ' . "\x28" . 'very useful
+			in making new friends' . "\x29" . ' - and perhaps what country you live in; if you do not wish
+			to be more specific than that.
+
+			We fully respect your right to privacy, and none of these items are necessary.
+			If you are new and do not know anybody here, they may help
+			you to make some new and interesting friends.
+
+			If you ever want to delete your account, you can do so at %1$s/settings/removeme
```

Comment:
We should remove the url part from the translation.

```suggestion
			If you ever want to delete your account, you can do so at %5$s
```